doc/source/data/examples/index.rst

```diff
@@ -0,0 +1,51 @@
+.. _datasets-examples-ref:
+
+========
+Examples
+========
+
+.. tip:: Check out the Datasets :`User Guide <data_user_guide>` To learn more about
+  Datasets' features in-depth.
+
+.. _datasets-recipes:
+
+Basic Data Processing Examples
+------------------------------
+
+Ray Datasets is a powerful engine for data processing that supports multiple data
+modalities and types. Here you will find a few end-to-end examples of some basic data
+processing with Ray Datasets.
+
+.. panels::
+    :container: container pb-4
+    :column: col-md-4 px-2 py-2
+    :img-top-cls: pt-5 w-75 d-block mx-auto
+
+    ---
+    :img-top: /images/taxi.png
```

Comment:
Yep this is a holdover from when I thought that this link pointed to `doc/source/data/images` rather than `doc/source/images`, let me remove that file.